static/styles/settings.scss

```diff
@@ -1577,40 +1620,19 @@ input[type=checkbox] {
     width: auto;
 }
 
-#account-settings,
-#admin-human-form {
```

Comment:
This got renamed in a commit you merge conflicted with, and it looks like you resolved it incorrectly, changing this to the old `#user-info-form-modal {`.